docs/server/upgrading.rst

```diff
@@ -144,20 +144,6 @@ To perform the upgrade follow the next steps:
     (env)$ pootle setup
 
 
-* If you are upgrading from Pootle 2.7.0 or later you also have to regenerate
-  the quality checks descriptions.
```

Comment:
The quality checks are part of the package.  You should never have to regenerate them on upgrade.